### DISCUSSION TITLE
Migrate new Blitz version

### DISCUSSION THREAD

**Discussion by levu304** [blitz-js/blitz#3049] [Q&A] [answered]
Hi, i'm currently using v0.43.0, now it's v0.44.3. Is there any documents about migration every upgrade?

**Comment by IRediTOTO** [comment]
I just did upgrade,  everything look fine :) Just commit old code and try to upgrade. If not work just need roll back :D

**Comment by beerose** [comment, marked as answer]
Hi @levu304, there are no breaking changes when upgrading from 0.43.0 to 0.43.3. If there are any, we usually include a note in the changelogs (https://github.com/blitz-js/blitz/releases) and a guide on how to migrate.
